app/src/main/java/io/awaken/ui/connections/ConnectionViewHolder.kt

```diff
@@ -0,0 +1,109 @@
+package io.awaken.ui.connections
+
+import android.app.AlertDialog
+import android.content.Intent
+import android.util.Log
+import android.view.View
+import android.widget.ImageButton
+import android.widget.ImageView
+import android.widget.TextView
+import android.widget.Toast
+
+import com.google.android.material.snackbar.Snackbar
+
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+import androidx.core.view.ViewCompat
+import androidx.recyclerview.widget.RecyclerView
+import io.awaken.R
+import io.awaken.data.database.ConnectionDatabaseHelper
+import io.awaken.data.network.Wake
+import io.awaken.ui.utils.AnimatedRecyclerView
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+
+/**
+ * @author Tyler Wong
+ */
+class ConnectionViewHolder internal constructor(view: View, recyclerView: AnimatedRecyclerView, private val mRefresher: ConnectionRefresher) : RecyclerView.ViewHolder(view) {
```

Comment:
I think we can move `internal` to before `class` and remove the `constructor` keyword.